lte/gateway/python/magma/mobilityd/ip_address_man.py

```diff
@@ -139,7 +140,8 @@ def __init__(self,
 
         self._recycle_timer = None  # reference to recycle timer
         self._recycling_interval_seconds = recycling_interval
-        self.static_ip_enabled = config.get('static_ip_enabled', False)
+        self.static_ip_enabled = config.get('static_ip_enabled',
```

Comment:
proto structs return the default value of the field if it's not specified so you can just do config.static_ip_enabled